pkg/app/managedisc/discovery_manager.go

```diff
@@ -0,0 +1,67 @@
+package managedisc
```

Comment:
I agree that a package name change is necessary. Not sure if `managedisc` is really a good name. We just recently changed from `appdisc` to `updatedisc` but I think under these circumstances `appdisc` may be better.

Reply:
Done!